internal/route/routes.go

```diff
@@ -73,14 +70,19 @@ func HandleIndex(w http.ResponseWriter, r *http.Request) {
 func HandleTodos(w http.ResponseWriter, r *http.Request) {
 	tpl := template.Must(template.ParseFiles("web/template/todo.html"))
 	vars := mux.Vars(r)
-	listID := vars["listId"]
+	listID, ok := vars["listId"]
+	if !ok {
+		log.Println("[error] listId not found")
+		w.WriteHeader(http.StatusNotFound)
+		return
+	}
```

Comment:
Just a reminder an example that when accessing maps, you have a second return value that you can use to make sure it was actually in the map.

Might not be necessary here since the URL wouldn't match without a `listId` but an example nonetheless.